### PR TITLE
i18n: add react-localization support for strings

### PR DESCRIPTION
**Pull request by islander**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change

## PR Type

What kind of change does this PR introduce?

- [x] Feature

## Fixes

Issue Number: https://github.com/btzr-io/Villain/issues/92

## What is the new behavior?
Now you can import `Localization` for translated strings.

## Other information
I've tried to implement this using existent `src/locales/messages*.js` files, don't know if it's correct decision. See `example/index.js`. Also, added Russian translation.

### CODE REVIEW

File: examples/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import Localization from '../src/localize'
```
Comment (btzr-io) [inline examples/index.js]: Use alias:
``` import Localization from '@/src/localize' ```

File: src/locales/messages.json [hunk 2]
```diff
@@ -9,5 +10,6 @@
   "theme.light": "Light theme",
   "theme.dark": "Dark theme",
   "zoom.in": "Zoom in",
-  "zoom.out": "Zoom out"
+  "zoom.out": "Zoom out",
+  "example.header": "Example test!"
```
Comment (btzr-io) [inline src/locales/messages.json]: is this required ?

Comment (btzr-io) [reply src/locales/messages.json]: `example.header`

Comment (islander) [reply src/locales/messages.json]: @btzr-io It's for demonstration purpose, see `src/example/index.js`

**Comment by btzr-io** [comment]
@islander Nice, great contribution, thanks :v:

**Comment by btzr-io** [comment]
@islander is not working, can you take another look ?

**Comment by islander** [comment]
@btzr-io you've fixed it here https://github.com/btzr-io/Villain/commit/b3d2e5b1a4a490beb1f87c66650ff20760e62fb1, or still need help?

**Comment by btzr-io** [comment]
@islander its working now but I'm not sure if thats the right implementation, feel free to improve it :v:

**Pull request merged**